docs/docs/advanced/setNativeProps.mdx

```diff
@@ -0,0 +1,85 @@
+---
+sidebar_position: 7
+---
+
+# setNativeProps
+
+`setNativeProps` lets you directly update component properties from the [UI thread](/docs/fundamentals/glossary#ui-thread).
```

Comment:
1. Let's add "imperatively" somewhere in the description
2. Please reference the original `setNativeProps` from RN.
3. Let's move the info about UI thread to a separate infobox.